workshop/buildspec-java.yaml

```diff
@@ -0,0 +1,50 @@
+version: 0.2
+
+
+env:
+    variables:
+      IMAGE_TAG : "v1.4"
+      AWS_DEFAULT_REGION: "us-west-2"
+      JAVA_IMAGE_REPO_NAME: "appmod-java-ecr"
+      
+phases:
+  install:
+    runtime-versions:
+      python: 3.11
+      
+  pre_build:
+    commands:
+      - echo "STARTING PYTHON INSTALLATION"
+      - pip3 install git-remote-codecommit
+      - echo Logging in to CodeCommit
+      - mkdir /opt/gradle
+      - wget -c https://services.gradle.org/distributions/gradle-4.2.1-bin.zip
+      - unzip -d /opt/gradle gradle-4.2.1-bin.zip
+      - export PATH=$PATH:/opt/gradle/gradle-4.2.1/bin
+      - yum -y install docker java-1.8.0
```

Comment:
can we use the latest LTS release of java? It is appmod and java 8 is getting close to a decade ago.
Gradle latest version is 8.4 and we are using 4.2 for some reason. 

Let's discuss if needed